### DISCUSSION TITLE
Feature request: allow audio passthrough when AG plugin is disconnected from AG server.

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#770] [Ideas]
It would be nice to allow audio to passthrough when the AG plugin is disconnected from AG server. I see the case where I would like to listen to the project without DSP. Needing to start the AG server system just to listen to a project is a bit of tedious.
Let me know what you think.
Thanks!

**Comment by Fittkenny** [comment]
Great idea!

**Comment by emack999** [comment]
@jaacMusic That's more an issue with whatever DAW or plugin host you are using.  For instance, in [Gig Performer](https://gigperformer.com/), the signal goes through an AG plugin that has no server connected just fine.

**Reply by jaacMusic** [reply]
At least for Cakewalk audio is not passed through, I assumed for some other DAW it would be the same.

**Reply by Henri75** [reply]
Same for Cubase.
And it would be usefull too :)

**Comment by apohl79** [comment]
Created a feature request and will add this
